requirements/conda-3.4.yml

```diff
@@ -20,4 +20,4 @@ dependencies:
 - wheel=0.29.0=py34_0
 - pip:
   - pyvisa==1.8
-  - pytest-qt==2.2.1
```

Comment:
I'm surprised that these changes continue to show as part of the diff after merging #191.

Reply:
Possibly this only gets refreshed if a new commit comes in?